code-organization.md

```diff
@@ -237,3 +238,61 @@ But what it gives us is extreme consistency by the most correct thing
 also the easiest.
 
 For a summary, please scroll back up and see the TL;DR section.
+
+## Inheritance
+
+We use a functional approach to inheritance, in this style:
+```
+var animal = function(options) {
+  var self = {},
+      location = 0,
+      speed = 5;
+  self.name = options.name;
+  
+  self.run = function(time) {
+    self.location += time * speed;
+  };
+  
+  self.getLocation = function() {
+    return self.location;
+  }
+  
+  return self;
+};
+
+var bear = animal({ name: 'Oso' });
+bear.run();
+// bear.name => "Oso"
+// bear.getLocation() => 5
```

Comment:
I think you forgot to provide the `time` parameter to `bear.run()`, so this will actually be `NaN`.

Same applies to other `.run()` and `.fly()` calls below.